src/Tribe/Views/V2/Rest_Endpoint.php

```diff
@@ -167,7 +167,7 @@ public function register() {
 		return register_rest_route( static::ROOT_NAMESPACE, '/html', [
 			// Support both GET and POST HTTP methods: we originally used GET.
 			'methods'             => [ Server::READABLE, Server::CREATABLE ],
-			 // @todo  Make sure we do proper handling of cache longer then 12h.
+			 // @todo [BTRIA-600]:  Make sure we do proper handling of caches longer then 12h.
```

Comment:
* :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 13

<sub>hash: 880854b8921dbaa68bc4fc9b41891288</sub>